### PR TITLE
Add F3D to C++ projects

### PR DESCRIPTION
**Pull request by mwestphal**
More info about F3D: https://f3d.app

### CODE REVIEW

**Review by MicahWSmith — APPROVED** [review]

**Comment by mwestphal** [comment]
@MicahWSmith should we merge this  ?

File: data.json [hunk 1]
```diff
@@ -157,6 +157,15 @@
             ],
             "description": "Easy-to-use open source robotics manipulation platform for developing commercial applications, prototyping designs, and benchmarking algorithms."
         },
+        {
+            "name": "F3D",
+            "link": "https://github.com/f3d-app/f3d",
+            "label": "good-first-issue",
```
Comment (sammyhori) [inline data.json:163]:
```suggestion
            "label": "good first issue",
```

Comment (sammyhori) [reply data.json:163]: otherwise, all looks fine

**Comment by sammyhori** [comment]
That's fair, apologies, I was trying to quickly work through PRs, closed about 115 that day

**Comment by mwestphal** [comment]
Thats fair as well :)

**Pull request merged**